study.md

```diff
@@ -27,17 +27,17 @@ it's also not a rendered page.
 Please name two things the DOM is not.
 
 ```md
-<!-- your answer here -->
+The DOM is not the HTML we write and it is not the HTML that you can see when you "view source" on a web page.
 ```
 
 Is the information inside your Dev Tools the DOM? Why or why not?
 
 ```md
-<!-- your answer here -->
+The information in my dev tools is the DOM because it is created from the HTML I wrote but perfected by the computer and can be manipulated by javascript that I write.
```

Comment:
The DOM is an interface that represents HTML documents in a tree structure.  What you see in your web browser's developer tools is a visual representation of the browser's implementation of that interface, but not the DOM itself.